crates/net/eth-wire/src/types/broadcast.rs

```diff
@@ -439,10 +441,9 @@ impl Decodable for NewPooledTransactionHashes68 {
     }
 }
 
-/// Interface for handling announcement data in filters in the transaction manager and transaction
-/// pool. Note: this trait may disappear when distinction between eth66 and eth68 hashes is more
-/// clearly defined, see <https://github.com/paradigmxyz/reth/issues/6148>.
-pub trait HandleAnnouncement {
+/// Interface for handling mempool message data. Used in various filters in pipelines in
+/// `TransactionsManager` and in queries to `TransactionPool`.
+pub trait HandleMempoolData {
```

Comment:
can even drop `Handle`?
```suggestion
pub trait MempoolData {
```

Reply:
I prefer to use imperatives for traits and reserve non-agent nouns for types https://github.com/rust-lang/api-guidelines/discussions/28